agent/util-scripts/pbench-tool-meister

```diff
@@ -374,15 +317,18 @@ class ToolMeister(object):
         self.state = "start"
         self._state_trans = {
             "start": {"next": "stop", "action": self.start_tools},
-            "stop": {"next": "send", "action": self.stop_tools},
-            "send": {"next": "start", "action": self.send_tools},
+            "stop": {"next": "start", "action": self.stop_tools},
+            "send": {"next": None, "action": self.send_tools},
```

Comment:
No longer very "state machiney" since there's no managed transition into "send" state. But this seems to support the idea that the tool meister is going start/stop/start/stop/start/stop and that the send operation is completely decoupled from the cycle. More of an discrete operation than a state...